### ISSUE TITLE
Configurable Jira Ticket Format Parsing?

### ISSUE THREAD

**Issue by kyledgreen** [jgillick/jira-changelog#6]
Hi,

I'd like to use this but we've been putting the ticket in our commit messages as "XX-1234: Commit Message" instead of surrounding it with square brackets.

Is it possible to configure the way it determines the ticket id?

**Comment by jgillick** [comment]
Yes, you can do this in your configuration file with the `ticketIDPattern` option:
https://github.com/jgillick/jira-changelog/blob/master/changelog.config.js#L24

So, in your case, you'd probably want it to be: `ticketIDPattern: /([A-Z]+\-[0-9]+)/i` or, if you want to include the colon after the JIRA ticket number: `ticketIDPattern: /([A-Z]+\-[0-9]+):/i`

**Comment by jgillick** [comment]
See the Configuration section of the README for more information about how to setup the config file:
https://github.com/jgillick/jira-changelog#configuration

Let me know if this helps.

**Comment by kyledgreen** [comment]
I will give that a try, thank you!
